provision-runners/action.yaml

```diff
@@ -0,0 +1,60 @@
+name: "Provision EC2 Runner"
+description: "Provision a GitHub Action runner in AWS"
+inputs: 
+  aws_access_key_id: 
+    description: 'Use a secret - this account needs "ec2-admin" permissions'
+    required: true
+  aws_secret_access_key: 
+    description: 'Use a secret - this account needs "ec2-admin" permissions'
+    required: true
+  runner_admin_token: 
+    description: "Use a secret. Token used to register a github action runner. If D6E Automaton token make sure it is added to the repo with Admin powers."
+    required: true
+  region: 
+    description: 'aws region for the runner. Default us-east-1'
+    required: false
+    default: 'us-east-1'
+  ami_id: 
+    description: "ami ID that will run on the mac2 instance. Must be associated with the Host Resource Group license."
+    required: true
+  ami_home_directory:
+    description: "the home directory for users on this AMI. Ex: /home"
+    required: true
+    default: /home
+  ami_user: 
+    description: "the automatically configured user on the ami. Usually ec2-user"
+    required: true
+  instance_type: 
+    description: "the instance type to spin up in the Host Resource Group"
+    required: true
+    default: 'mac2.metal'
+  host_resource_group_arn: 
+    description: "the arn of the HRG that will automate setup of the dedicated hosts for the runners"
+    required: false
+    default: ""
```

Comment:
What's the behavior when the `host_resource_group_arn` is empty?

Reply:
If the HRG is empty the instance will be treated as a regular on-demand instance. For a Mac that would not be allowed.